apis/types/log_config.go

```diff
@@ -0,0 +1,120 @@
+// Code generated by go-swagger; DO NOT EDIT.
+
+package types
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	"encoding/json"
+
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/swag"
+	"github.com/go-openapi/validate"
+)
+
+// LogConfig The logging configuration for this container
+// swagger:model LogConfig
+
+type LogConfig struct {
+
+	// config
+	Config map[string]string `json:"Config,omitempty"`
+
+	// type
+	Type string `json:"Type,omitempty"`
+}
```

Comment:
In moby, the LogConfig has a type of 

```
// LogConfig represents the default log configuration.
// It includes json tags to deserialize configuration from a file
// using the same names that the flags in the command line use.
type LogConfig struct {
	Type   string            `json:"log-driver,omitempty"`
	Config map[string]string `json:"log-opts,omitempty"`
}
```

The json naming of the fields are different, `log-driver` rather than `Type`, `log-opts` rather than `Config`.
Do you think we need to keep compatible? @fuweid

Reply:
Make senses. Will update it.